integrations/sonarqube/main.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from sonarqube_integration.sonarqube_client import SonarQubeClient
+import httpx
+
+
+class ObjectKind:
+    PROJECTS = "projects"
+    ISSUES = "issues"
+    ANALYSIS = "analysis"
+
+
+# Initialize the SonarQubeClient instance
+sonar_client = SonarQubeClient(
```

Comment:
Initialize it in here can cause problems in the CLI, please take a look how we intialized PagerDuty client and make it similar here
https://github.com/port-labs/ocean/tree/main/integrations/pagerduty